pwa/src/templates/templateParts/schemaForm/SchemaFormTemplate.tsx

```diff
@@ -57,6 +57,8 @@ export const SchemaFormTemplate: React.FC<SchemaFormTemplateProps & ReactHookFor
     setSimpleProperties([]);
     setComplexProperties([]);
 
+    if (!schema.properties) return;
```

Comment:
Is this intentional? Just checking 👍🏼

Reply:
It is, without this line objects with no properties cannot be displayed and will cause an error.
And we still want to display an object even if the object does not have any properties.